README.md

```diff
@@ -78,9 +78,7 @@ deno run https://deno.land/std/examples/welcome.ts
 Or [setup a simple HTTP server](https://examples.deno.land/http-server):
 
 ```ts
-import { serve } from "https://deno.land/std@0.182.0/http/server.ts";
-
-serve((_req) => new Response("Hello, World!"));
+Deno.serve((_req) => new Response("Hello, World!"));
```

Comment:
Do we want to keep `_req` here?

Reply:
yes; it at least gives a general idea of what the first & most commonly used argument is without having to be specific

Reply:
Makes sense. I thought that might be it.